client/app/kubernetes/service/kubernetes-service.html

```diff
@@ -12,6 +12,18 @@
         <!--Cluster Information-->
         <div class="col-md-4 col-xm-12" data-ng-if="$ctrl.cluster">
             <oui-tile class="h-100" data-heading="{{:: 'kube_service_cluster_information' | translate }}">
+
+                <oui-tile-definition data-term="{{:: 'kube_service_name' | translate }}">
+                    <oui-tile-description data-ng-bind="$ctrl.cluster.name"></oui-tile-description>
+                    <oui-action-menu data-compact data-align="end">
+                        <oui-action-menu-item
+                            data-text="{{::'common_edit'|translate}}"
```

Comment:
FYI `data-text` attribut is deprecated (ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu#deprecated)